### PR TITLE
Readme updates

### PR DESCRIPTION
**Pull request by ultratin**
Updated readme with basic installation instruction and how to start the project.

Also added webpack-dev-server as part of the startup process for foreman

### CODE REVIEW

**Comment by elliotcm** [comment]
Looks like you were partially pipped at the post by https://github.com/SmartCasual/jingle-jam/pull/71

There's some parts of this that I'd still like to keep, so I'll rebase it once the other PR is in.  Thanks!

**Comment by ultratin** [comment]
@elliotcm sure thing. Happy to help!

**Comment by gormster** [comment]
@ultratin ah, you gotta check the open PRs! sorry mate ;)

File: Procfile [hunk 1]
```diff
@@ -1,2 +1,3 @@
 web: bin/rails server -p ${PORT:-5000} -e $RAILS_ENV
 worker: bundle exec sidekiq
+webpack-dev-server: bin/webpack-dev-server
```
Comment (elliotcm) [inline Procfile:3]:
```suggestion
webpack-dev-server: bin/webpack-dev-server

```

**Review by elliotcm — CHANGES_REQUESTED** [review]

**Review by elliotcm — APPROVED** [review]

**Pull request merged**